.github/workflows/api-deploy-production-ecs.yml

```diff
@@ -47,24 +47,9 @@ jobs:
         needs: deploy-production-ecs
 
         steps:
-            - name: Cloning repo
-              uses: actions/checkout@v2
+            - name: Run E2E tests against production
+              uses: ./.github/actions/e2e-tests
               with:
-                  fetch-depth: 0
-
-            - name: Test with Chromedriver
-              working-directory: frontend
-              env:
-                  E2E_TEST_TOKEN_STAGING: ${{ secrets.E2E_TEST_TOKEN }}
```

Comment:
@dabeeeenster I imagine this was the issue that caused the previous failure - we were setting the _STAGING token but trying to retrieve the prod token.

Reply:
Ah yes